components/webgpu/swapchain.rs

```diff
@@ -350,6 +381,9 @@ impl crate::WGPU {
         let mut webgpu_contexts = self.wgpu_image_map.lock().unwrap();
         let context_data = webgpu_contexts.get_mut(&context_id).unwrap();
 
+        let presentation_id = context_data.next_presentation_id();
+        assert!(context_data.check_and_update_presentation_id(presentation_id));
```

Comment:
This is trivially true, so not useful as an assertion.

Reply:
Will remove it.